.github/workflows/pr.yaml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
   pull_request:
+    types: [ unlabeled, labeled, opened, synchronize, reopened ]
```

Comment:
Does this run on PRs without any labels now ? 😨

Reply:
yes, these are the events that actually trigger the run. So on opening, synchronizing, labeling, unlabeling, etc.

Reply:
Without any label runs the exact same thing that used to run, but this PR adds the functionality that when the `all-distro` label is around in addition it also runs the CI for other distros. Running `arm64` tests by including the `arm64` label was already possible. 

The change here on this line only express under which events the workflow is re-triggered. The default value is `opened`, `synchronize` and  `reopened`. Then I added `labeled` and `unlabeled` so it also reacts on label changes and there is no need to add the desired labels before hand at PR creation.

I think probably a better approach would be moving the `arm64` tests out of the PR scope and run them only on demand by manual workflow and/or based on some schedule, but not on PRs.